src/server_manager/web_app/ui_components/outline-server-settings.js

```diff
@@ -299,7 +299,7 @@ Polymer({
     }
     // Fire signal if name has changed.
     if (newName !== this.initialName) {
-      this.fire('ServerRenameRequested', {newName});
+      this.fire('ServerRenameRequested', {newName, displayServer: this._makeDisplayServer()});
```

Comment:
To make it more explicit about which server to rename, we would pass along the information in the event.

We're passing the whole _DisplayServer_ object in this PR, but this can be simplified to just the server ID.